src/FINDOLOGIC/Export/Exceptions/XMLSchemaViolationException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace FINDOLOGIC\Export\Exceptions;
+
+use RuntimeException;
+
+/**
+ * Thrown in case an XML export page does not conform to the schema.
+ */
+class XMLSchemaViolationException extends RuntimeException
+{
+    public function __construct(array $validationErrors)
+    {
+        parent::__construct('XML schema validation failed: ' . implode(';', $validationErrors));
```

Comment:
Use a formated string here.

Reply:
I don't see any value in using `sprintf` if there is only one value to interpolate, and this one value is appended. The concatenation operator makes more sense in my eyes.